src/ucp/rndv/proto_rndv.c

```diff
@@ -331,7 +348,8 @@ ucs_status_t ucp_proto_rndv_rts_init(const ucp_proto_init_params_t *init_params)
         .unpack_time         = UCS_LINEAR_FUNC_ZERO,
         .perf_bias           = context->config.ext.rndv_perf_diff / 100.0,
         .mem_info.type       = init_params->select_param->mem_type,
-        .mem_info.sys_dev    = init_params->select_param->sys_dev
+        .mem_info.sys_dev    = init_params->select_param->sys_dev,
+        .ctrl_msg_name       = "RTS"
```

Comment:
wdyt about adding `UCP_PROTO_RNDV_RTS_NAME`? - same as we have for `UCP_PROTO_RNDV_ATS_NAME`/`UCP_PROTO_RNDV_ATP_NAME`